### ISSUE TITLE
Grid columns for selecting project skipped with Tab but not with Shift-Tab

### ISSUE THREAD

**Issue by zkat** [NuGet/Home#10202]
## Details about Problem

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe): PMUI

NuGet version (x.x.x.xxx):

dotnet.exe --version (if appropriate):

VS version (if appropriate):  16.4

OS version (i.e. win10 v1607 (14393.321)):

Worked before? If so, with which NuGet version:

## Detailed repro steps so we can see the same problem

This is a problem for both NVDA and Narrator.

Right-panel -> Description field is not accessible by tabbing, and therefore isn't readable by any screen-reader.

Packages list, it only reads the Package Name, and does not read the Description. I'm not sure that we would want it to by default. But right now, there's no way to read a description at all.

**Comment by donnie-msft** [comment]
@zkat where was this issue reported from so I can see the full context?

**Comment by donnie-msft** [comment]
@zkat I'm closing this as I still don't know what may have lead you to reporting this. Please reopen if you find something documenting the problem and the desired behavior.
Accessibility insights passes all checks:
![image](https://user-images.githubusercontent.com/49205731/103929233-036ddb80-50eb-11eb-8910-8e00158eb3f6.png)
![image](https://user-images.githubusercontent.com/49205731/103929305-1d0f2300-50eb-11eb-9840-062b850fcfde.png)

**Comment by zkat** [comment]
...yeah I have no memory of why I made this. Closing is fine, sorry.
